### ISSUE TITLE
'Set-ExecutionPolicy' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by Sammeeey** [angular/angular#50088]
### Describe the problem that you experienced

In [*Install the Angular CLI*](https://angular.io/guide/setup-local#install-the-angular-cli) there's a hint that instructs to run the following command on Windows:
```
Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned
```
If I do so, I encounter the following error message: 
```
'Set-ExecutionPolicy' is not recognized as an internal or external command,
operable program or batch file.
```

The installation of the Angular CLI seems to work anyway for me (using [CMDer](https://cmder.app/) on Windows 10).

Angular was already installed properly after running `npm install -g @angular/cli` (tested by running `ng version`).

However `npm install -g @angular/cli` gives me this response:
```
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs

changed 256 packages in 25s

33 packages are looking for funding
  run `npm fund` for details
```

But [people say that the *WARN* isn't a problem](https://stackoverflow.com/questions/74337166/npm-warn-deprecated-npmcli-move-file2-0-1-this-functionality-has-been-moved-t).

---
## Long Story short
The hint box instructing to run `Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned` seems to be superfluous (at least for me on Win10).

Shall we remove it from the docs?

**Comment by JoostK** [comment]
This depends on the shell that is being used. In `cmd` this is an unknown command, but using `PowerShell`  it may be needed (depending on various factors, e.g. whether Windows Development Mode is enabled).

This was requested to be documented in #44598 and implemented in #45576. I think that comment can use some clarification to mention this is needed for users running PowerShell.

**Comment by Sammeeey** [comment]
True, I just experienced that it in fact doesn't work in PowerShell.
